searches/views.py

```diff
@@ -36,8 +36,7 @@ def get(self, request, query, sem_name, year):
         school = request.subdomain
         sem = Semester.objects.get_or_create(name=sem_name, year=year)[0]
 
-        # grab first 10 results
-        course_matches = search(request.subdomain, query, sem).distinct()[:10]
+        course_matches = search(request.subdomain, query, sem).distinct().order_by("id")
```

Comment:
Do you think there's some way to get rid of `distinct()` here?

Reply:
I will look into this

Reply:
Also, is the order_by necessary? Would it be the same if it weren't there?

Reply:
If I don't do an order_by, Django throws a warning: `pagination may yield inconsistent results with an unordered object_list:`

Reply:
Weird that it isn't automatically sorted by id but okay

Reply:
> I will look into this

Let me know what you find here and we can merge afterwards

Reply:
@JJamesWWang Thye added `distinct()` here because without it, there will be duplicated courses:
![image](https://user-images.githubusercontent.com/37493948/218933594-03129a1f-6546-49ab-a375-433d1d380cf1.png)
And the reason that there are duplicated courses is that they are performing an inner join on sections. 
`Course.objects.filter(
        Q(school=school) & course_name_contains_query & Q(section__semester=semester)
    )`
Each course is duplicated by the # of sections it has. They need to join on sections because courses persist across semesters:
![image](https://user-images.githubusercontent.com/37493948/218934034-36584785-4900-469f-b4ce-f6fa5db312f6.png)
So joining the sections would return courses that have sections in the given semester. The consequence is that each course is duplicated by the number of sections it has so they used `distinct()` to eliminate the duplicates.
This is my understanding for now and I will try to think of a way to query the db so we don't use `distinct()`